src/microbots/MicroBot.py

```diff
@@ -105,9 +108,28 @@ def run(self, task, max_iterations=20, timeout_in_seconds=200) -> BotRunResult:
             result=None,
             error="Did not complete",
         )
-        logger.info("%s TASK STARTED : %s...", LogLevelEmoji.INFO, task[0:15])
+        print(f"\n{'='*80}")
+        print(f"🚀 TASK STARTED: {task}")
+        print(f"{'='*80}")
```

Comment:
Let's convert all `print` statements into `INFO` log statements. To avoid being unresponsive in the console while running, add a `WARNING` log to mention `Log Level is above INFO. So, no output will be printed here during normal operation. Please wait patiently`.

It gives us better control over the output and log stream. Particularly useful to run in quiet mode.